docs/rest_api.md

```diff
@@ -97,3 +108,10 @@ Full Api spec:
             "nics": ["ipmi", "pxe", "public", ...]
         }
     [GET] /headnode/<headnode>
+
+    show_network <network> ->
+        {
+            "name": "my-net",
+            "access": "my-proj", # optional; absence means a public network.
+            "creator": "my-proj" # either a project or "admin"
```

Comment:
Is there a compelling reason to include "creator"? Once something is exported, it's difficult to take back later.

Perhaps additional information should be added for admins? (like VLAN id)